### PR TITLE
Add malware disclaimer

### PR DESCRIPTION
**Pull request by egibs**
Relates to: #727 

This PR adds a disclaimer explaining why malcontent is seen as malicious by other programs (Yara detecting Yara 🔁).

### CODE REVIEW

**Review by miketheman — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -35,6 +35,28 @@ malcontent has 3 modes of operation:
 
 malcontent is at its best analyzing programs that run on Linux. Still, it also performs admirably for programs designed for other UNIX platforms such as macOS and, to a lesser extent, Windows.
 
+## ⚠️ Malware Disclaimer ⚠️
+
+Due to how malcontent operates, other malware scanners can detect malcontent as malicious. As a general rule of thumb, programs that leverage Yara rules will match other programs that use the same rules due to their strings looking for problematic behaviors.
+
+While not exhaustive, here's an example list of how other scanners see malcontent (based on [this](https://www.virustotal.com/gui/file/b6f90aa5b9e7f3a5729a82f3ea35f96439691e150e0558c577a8541d3a187ba4/detection) VirusTotal scan:
```
Comment (tstromberg) [inline README.md:42]: I think this exhaustive list is going to go stale very very quickly. Can we just remove it and point people to the example issue?

File: README.md [hunk 2]
```diff
@@ -35,6 +35,28 @@ malcontent has 3 modes of operation:
...
+- Kaspersky: `HEUR:Trojan-PSW.OSX.Amos.n`
+- MaxSecure: `Trojan.Malware.121218.susgen`
+- Rising: `Backdoor.JokerSpy/OSX!1.E753 (CLASSIC)`
+- Sangfor `Engine Zero: HackTool.Win32.Template_Py_v3_3_to_v4_x.uwccg`
+- SentinelOne (Static ML): `Static AI - Malicious Mach-O`
+- WithSecure: `Malware.OSX/GM.Joker.DS`
+
+Elastic's Agent has also historically detected malcontent because of this: https://github.com/chainguard-dev/malcontent/issues/78
+
+While not a permanent solution, running malcontent with `--third-party=false` can reduce these false positives. Writing more targeted rules can also help.
```
Comment (tstromberg) [inline README.md:58]: How about something like "reduce false-positive detections that occur at runtime". 

--third-party=false won't help the aforementioned detections as they are all static. 

I don't think the "Writing more targeted rules can also help" is particularly helpful to our users.

**Review by tstromberg — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (egibs) [reply README.md]: Removed in [`5eedaaa` (#728)](https://github.com/chainguard-dev/malcontent/pull/728/commits/5eedaaa418e7f6d5aa0e162e838e6a9722537ee3).

File: README.md (hunk 2 above)
Comment (egibs) [reply README.md]: I ended up removing this entirely in [`5eedaaa` (#728)](https://github.com/chainguard-dev/malcontent/pull/728/commits/5eedaaa418e7f6d5aa0e162e838e6a9722537ee3).

**Review by stevebeattie — APPROVED** [review]
LGTM, thanks!

**Pull request merged**